website/app/controllers/application.js

```diff
@@ -28,17 +29,52 @@ export default class ApplicationController extends Controller {
     return this.router.currentURL.split('/')[1];
   }
 
+  resetSidebar() {
+    this.isSidebarVisibleOnSmallViewport = false;
+    if (!this.fastboot.isFastBoot) {
+      document.body.classList.remove('isSidebarVisibleOnSmallViewport');
+      document.body.classList.remove('isSidebarInRenderTree');
+      if (this.runNext) {
```

Comment:
maybe we could pair on this but there should be a more straightforward way to interact with the run loop without needing to do this sort of thing manually

Reply:
> so we need to use two "ticks" in the loop

My point with the comment is that my impression/recollection is that there is (or should be) a cleaner way to accomplish this in Ember than manually tracking runloop events like this.